tests/unit/src/test/scala/tests/TimerSuite.scala

```diff
@@ -13,17 +13,28 @@ object TimerSuite extends BaseSuite {
       assertNoDiff(obtained, expected)
     }
   }
+
+  // Decimal separator of the default locale of the machine.
+  val decimalPointSeparator = {
+    import java.text.DecimalFormatSymbols
+    import java.util.Locale
+    new DecimalFormatSymbols(Locale.getDefault).getDecimalSeparator()
+  }
+
+  // Replaces the ('.') with the current decimal separator
+  def fixDPS(s: String) = s.replace('.', decimalPointSeparator)
```

Comment:
Instead of patching the output for tests here, can we instead hardcode the locale in `Timer.readableNanos`? That ensure we produce consistent output regardless of locale.

```scala
        new DecimalFormat("#.##s", new DecimalFormatSymbols(Locale.US))
          .format(partialSeconds)
```